docs/getting-started.rst

```diff
@@ -6,6 +6,13 @@ Getting started
 
 Install and get started with the :ref:`CrateDB Npgsql Plugin <index>`.
 
+.. NOTE::
+
+    Please note that this driver is not recommended with recent releases of
+    CrateDB >= 4.2, which works fine with the vanilla Npgsql driver. See
+    also :ref:`index`.
+
+
```

Comment:
```suggestion
CAUTION::

    For CrateDB versions 4.2 and above, we recommend that you use the `stock
    Npgsql driver` instead of this one.

    This software is a legacy plugin for older versions of CrateDB that lacked
    full compatibility with Npgsql.

```

you will then need to add this to the bottom of the file:

```
.. _stock Npgsql driver: https://www.npgsql.org/
```

Reply:
Thank you. I just added c51cdd9af70824.